libopenage/game_control.h

```diff
@@ -103,6 +135,32 @@ class ActionMode : public OutputMode {
 
 	// used for random type creation
 	rng::RNG rng;
+
+	std::tuple<int, int, int, int> resources;
```

Comment:
We decided to use an arbitrary amount of resources and resource types which are defined via the gamespec (later `nyan`). For now it's ok, but we should keep that flexibility in mind.